L1Trigger/Configuration/python/customiseSettings.py

```diff
@@ -2,6 +2,22 @@
 import os.path
 import FWCore.ParameterSet.Config as cms
 
+def L1TSettingsToCaloParams_2023_v0_4(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_4_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2023_v0_3(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_3_cfi")
+    return process
+
+def L1TSettingsToCaloParams_2023_v0_2(process):
+    process.load("L1Trigger.L1TCalorimeter.caloParams_2023_v0_2_cfi")
```

Comment:
Where are all these configs?
I can't see them in L1Trigger/L1TCalorimeter/python

Reply:
These are all present in the L1 offline fork. I've been requesting phase 1 L1 developers discontinue use of our generic L1 offline fork to do away with the difficulties of maintaining parallel forks and (un)duplicate information, and it looks like these never made the jump. I can open a PR to @bundocka 's branch to include these.

Reply:
@bundocka Could you either merge https://github.com/bundocka/cmssw/pull/4 or remove the v0_1 through v0_3 customization options?